lms/static/js/spec/search/search_spec.js

```diff
@@ -0,0 +1,489 @@
+define([
+    'jquery',
+    'sinon',
+    'backbone',
+    'js/common_helpers/template_helpers',
+    'js/search/views/SearchForm',
+    'js/search/views/SearchItemView',
+    'js/search/views/SearchListView',
+    'js/search/models/SearchResult',
+    'js/search/collections/SearchCollection',
+    'js/search/SearchRouter',
+    'js/search/SearchApp'
+], function(
+    $,
+    Sinon,
+    Backbone,
+    TemplateHelpers,
+    SearchForm,
+    SearchItemView,
+    SearchListView,
+    SearchResult,
+    SearchCollection,
+    SearchRouter,
+    SearchApp
+) {
+    'use strict';
+
+    describe('SearchForm', function () {
+
+        beforeEach(function () {
+            loadFixtures('js/fixtures/search_form.html');
+            this.form = new SearchForm();
+            this.onClear = jasmine.createSpy('onClear');
+            this.onSearch = jasmine.createSpy('onSearch');
+            this.form.on('clear', this.onClear);
+            this.form.on('search', this.onSearch);
+        });
+
+        it('prevents default action on submit', function () {
```

Comment:
I find it a bit odd to write a test for whether a particular method has been performed in the implementation. I think if there are tests that verify that the form works as desired then this test isn't necessary. Otherwise we'd need such a test on every single form throughout our code.